lib/fluent/plugin/buffer/chunk.rb

```diff
@@ -149,11 +152,31 @@ def open(&block)
           raise NotImplementedError, "Implement this method in child class"
         end
 
-        def write_to(io)
+        def write_to(io, options = {})
           open do |i|
             IO.copy_stream(i, io)
           end
         end
+
```

Comment:
There are no care about `chunk.read`, so that call returns raw value of compressed data.
Is it intended?
I think `chunk.read` (without any optional arguments) should return decompressed data.